### PR TITLE
[3.7.x] Convert installation template to html5 (also move inline scripts to js)

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
### Summary of Changes

Convert installation template to html5, also move inline scripts to js
### Testing Instructions
1. Code review
2. Apply patch in 3.7.x branch
3. Delete configuration.php
4. Install joomla as usual and confirm install works
### Documentation Changes Required

None.

### CODE REVIEW

**Comment by ghazal** [comment]
PHP 7.0.11
joomla 3.7dev (last version)
* already installed
configuration.php deleted
installation folder added
file index.php modified
*new install
file index.php modified

In both cases, I can't go past the first window (Configuration)
No JS error.

**Comment by ciar4n** [comment]
Similar story here, unable to proceed from step 1. Getting the following error in console when clicking 'Next'.


```
"NetworkError: 403 Forbidden - http://localhost/joomla-cms/installation/%3C?php%20echo%20JUri::current();%20?%3E"
```

**Comment by andrepereiradasilva** [comment]
my bad, sorry will correct

File: installation/template/index.php [hunk 1]
```diff
@@ -37,36 +40,138 @@
...
 JText::script('INSTL_PROCESS_BUSY');
 JText::script('INSTL_FTP_SETTINGS_CORRECT');
+
+// Add inline scripts
+$this->addScriptDeclaration("
+	jQuery(function()
+	{
+		// Delay instantiation after document.formvalidation and other dependencies loaded
+		window.setTimeout(function(){
+			window.Install = new Installation('container-installation', '<?php echo JUri::current(); ?>');
```
Comment (dgt41) [inline installation/template/index.php]: change this to:
```js
window.Install = new Installation('container-installation', '" . JUri::current() . "');

Comment (andrepereiradasilva) [inline installation/template/index.php]: already changed, i add changed in my local install but didn't commit it

File: installation/template/index.php [hunk 2]
```diff
@@ -37,36 +40,138 @@
...
 JText::script('INSTL_FTP_SETTINGS_CORRECT');
+
+// Add inline scripts
+$this->addScriptDeclaration("
+	jQuery(function()
+	{
+		// Delay instantiation after document.formvalidation and other dependencies loaded
+		window.setTimeout(function(){
+			window.Install = new Installation('container-installation', '" . JUri::current() . "');
+		}, 500);
```
Comment (Fedik) [inline installation/template/index.php]: hm, strange piece of code, it always was like that? 😄 
there no guarantee that all will be loaded in 500ms,
for this need to use `jQuery(window).load(function(){....})`

if it always was like that then nevermind 😄

**Comment by ciar4n** [comment]
Installing fine however the 'Site Offline' toggle is appearing as follows...

install-switch

File: installation/template/index.php (hunk 2 above)
Comment (mbabker) [inline installation/template/index.php]: > installation ~~js~~ IMHO needs some review ...

There, fixed that for you 😉

**Comment by andrepereiradasilva** [comment]
ok, please do not test will review all later again, since it seems there are some issues and i don't have much time right now.

**Comment by andrepereiradasilva** [comment]
should work now.

removed all inline js and moved it to installation.js

@Fedik @dgt41 @ciar4n please check and test

**Comment by ghazal** [comment]
Can't post from Issues (Error : Missing commit SHA).
Already Installed and fresh installed versions of joomla 3.7 both work alright.

**Comment by brianteeman** [comment]
@ghazal - I have done it for you - but if you refresh your browser cache then you should see the sha now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12444">issues.joomla.org/tracker/joomla-cms/12444</a>.</sub>

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on 748d4c1df98d1fddb9b805e44c636700d2c17ace<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12444">issues.joomla.org/tracker/joomla-cms/12444</a>.</sub>

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 748d4c1df98d1fddb9b805e44c636700d2c17ace<br /><br />All good! :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12444">issues.joomla.org/tracker/joomla-cms/12444</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12444">issues.joomla.org/tracker/joomla-cms/12444</a>.</sub>

**Pull request merged**